### DISCUSSION TITLE
RSS/Atom feed for new releases (all projects)?

### DISCUSSION THREAD

**Discussion by Chaz6** [community/community#22563] [New to GitHub]
<p>Is there a url to a RSS/Atom feed which contains new releases for all projects on GitHub? I do not want to subscribe to individual projects, but see every new release on GitHub in one place.</p>

**Comment by mpboom** [comment, marked as answer]
<p>You can find more information about feeds availible right here: <a href="https://developer.github.com/v3/activity/feeds/" rel="nofollow noopener">https://developer.github.com/v3/activity/feeds/</a></p>
